popup/popup.js

```diff
@@ -1,7 +1,7 @@
 
 window.onload = function() {
     document.getElementById('closeBtn').onclick = closeExtension;
-    document.getElementById('searchField').addEventListener('keyup', updateLocalStorage);
+    document.getElementById('searchField').addEventListener('input', updateLocalStorage);
```

Comment:
Oh I see this now. Strange, it only had this behavior initially for me. Input event works better, so we may just need to sacrifice a bit of performance. Can update later if we find a solution. Merging.